build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -55,6 +48,11 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 			required: true,
 			multiple: true,
 		}),
+		creds: Flags.string({
+			description: "Username and email id for GitHub auth",
+			required: true,
```

Comment:
You don't want to require these if you're going to set the git config settings in the command code. If you do that, then when testing locally, people will have to 

I think you're basically in a position where the repo has to be "configured properly" in order to use the command. That's totally fine. The question is where should that config happen? I think the best place is in the workflow itself.

Imagine two scenarios - running the command locally for testing and running in CI. When running locally, you would assume safely that the user has their repo configured with a username and email. When in CI, it's sort of like the bot running the command locally - so it's reasonable to expect "the bot" to configure their repo as well. So I think setting the username/email in the workflow before calling the flub command makes more sense and will be simpler code overall.